src/components/nodes/task/task.vue

```diff
@@ -111,6 +116,9 @@ export default {
     },
   },
   methods: {
+    resetCustomIconName() {
+      this.setCustomIconName('');
+    },
```

Comment:
This looks like it's the same `resetCustomIconName` method that's defined in the `customIcon` mixin and can safely be removed from here.